policyengine_us/parameters/gov/states/il/tax/income/exemption/personal.yaml

```diff
@@ -2,12 +2,22 @@ description: Illinois provides the following personal exemption allowance amount
 values:
   2021-01-01: 2_375
   2022-01-01: 2_425
+  2025-01-01: 2_050
```

Comment:
the 2024 value will take some research, need to calculate inflation manually since uprating doesn't currently have a either manually calculate the 2024 value from inflation or apply the cadence-based uprating per https://github.com/PolicyEngine/policyengine-core/pull/173

illinois basically just interrupted the normal inflation adjustments in 2023
```suggestion
  2023-01-01: 2_425 # Redundancy per legal code.
  2024-01-01: x # $2,050 plus COLA since 2011.
```

Reply:
> the 2024 value will take some research, need to calculate inflation manually since uprating doesn't currently have a either manually calculate the 2024 value from inflation or apply the cadence-based uprating per [PolicyEngine/policyengine-core#173](https://github.com/PolicyEngine/policyengine-core/pull/173)
> 
> illinois basically just interrupted the normal inflation adjustments in 2023

Ok based on these two https://www.bls.gov/news.release/archives/cpi_09152011.pdf and https://www.bls.gov/news.release/archives/cpi_09132023.pdf    the 2011 end of twelve month close (for august) was 3.8% but for 2023 was 3.7% so the target year doesn't exceed base year, so in that case the final value would just be 2050+0? because there is no excess
![image](https://github.com/PolicyEngine/policyengine-us/assets/72925608/13034b13-fc34-44c9-9288-0ce41e2871a6)

Reply:
Cpi value not inflation rate (which is y/y change in cpi)